crates/api_common/src/person.rs

```diff
@@ -51,15 +51,6 @@ pub struct Register {
   pub answer: Option<String>,
 }
 
-#[skip_serializing_none]
-#[derive(Debug, Serialize, Deserialize, Clone, Default)]
-#[cfg_attr(feature = "full", derive(TS))]
-#[cfg_attr(feature = "full", ts(export))]
-/// Fetches a Captcha item.
-pub struct GetCaptcha {
-  pub auth: Option<Sensitive<String>>,
-}
```

Comment:
I had to add these for ts-rs reasons iirc. But I might be able to handle that in the UIs.

Reply:
Wasnt it for websocket? ts-ts doesnt have anything to do for a nonexistant struct.